Getting Started with Splitter (#6266)
DevExpress/devextreme-documentation: concepts/05 UI Components/Splitter/00 Getting Started with Splitter/10 Configure Panes.md

> The following base options are available for a pane:

Reviewer: I'm not sure what you mean by "base option". I understand that's something "basic", but why these? Why "size" isn't a "base option"? And why "collapsedSize" is not in the same category as "collapsed" - they seem to be working together. 

 ```suggestion
The following options specify the Splitter state and available user interaction capabilities:
```

- [resizable](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#resizable) (default: `true`)    

> If true, a handle appears at the side of the pane. Drag the handle to adjust pane size.

Reviewer: Can we say "separator" instead of "the side of the pane"? We already mentioned "separators" and the word is clear enough anyway.

```suggestion
If true, a handle appears on the pane separator. Drag the handle to adjust pane size.
```

- [collapsible](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#collapsible) (default: `false`)    

> If true, an arrow appears on the handle. Click the arrow to close the pane.

Reviewer: ```suggestion
If true, an arrow appears on the handle. Click the arrow to collapse/expand the pane.
```

- [collapsed](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#collapsed) (default: `false`)    

> Specifies whether the pane is initially collapsed.

Reviewer: I assume I can change this value at any time? It's not necessarily the initial state? 

```suggestion
Specifies whether the pane is collapsed.
```

> All Splitter "size" properties depend on its [orientation](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation). If it is *'horizontal'*, "size" is the width. Otherwise, it is height. The following pane "size" properties can be specified:

Reviewer: ```suggestion
Use the following properties to specify pane size. Note that all these properties depend on Splitter [orientation](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#orientation). They specify width in   *'horizontal'* orientation and height in *'vertical'* orientation. 
```

- [size](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#size)    

> Initial pane width/height.

Reviewer: ```suggestion
Pane width/height.
```

- [minSize](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#minSize)    
Minimum width/height for a resizable pane.

- [maxSize](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#maxSize)    
Maximum width/height for a resizable pane.

- [collapsedSize](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#collapsedSize)    

> The width/height of a collapsible pane when collapsed.

Reviewer: ```suggestion
Pane width/height in the collapsed state.
```

> This tutorial explicitly specifies the following properties: **size**, **minSize**, **maxSize**, and **collapsible**. All panes are **resizable**, since this is a default behavior.

Reviewer 2: ```suggestion
This tutorial explicitly specifies the **size**, **minSize**, **maxSize**, and **collapsible** properties. All panes are **resizable** by default.
```
